backend/src/main/scala/bloop/BloopClassFileManager.scala

```diff
@@ -201,6 +201,34 @@ final class BloopClassFileManager(
 
   }
 
+  private def copyResources(
+      resources: List[AbsolutePath],
+      copyTo: AbsolutePath,
+      config: ParallelOps.CopyConfiguration
+  ): Task[Unit] = {
+    val (singleFiles, classpathEntries) =
+      resources.partition(path => path.exists && path.isFile)
```

Comment:
Are we sure there is no jar in here?

Reply:
Even if there is, we want to copy it since it's set as a resource. Though, I have never seen a case like this.

Reply:
Okay that's fair